src/gatherers/manifest.js

```diff
@@ -19,30 +19,37 @@
 const Gather = require('./gather');
 const manifestParser = require('../lib/manifest-parser');
 
-/* global document, XMLHttpRequest */
+/* global document, XMLHttpRequest, __inspect */
 
 function getManifestContent() {
+  function post(response) {
+    // __inspect is magically inserted by driver.evaluateAsync
+    __inspect(JSON.stringify(response));
```

Comment:
the `JSON.stringify`/`parse` stuff should be moved into `evaluateAsync` so audits don't have to worry about it (by having, like
```
(function() {
  function __returnResult(result) {
    inspect(JSON.stringify(result));
  }
})();
```
or whatever)